net/lp_rpc.proto

```diff
@@ -154,3 +157,56 @@ message NotifySegment {
     int64 taskId   = 16;
     bytes profiles = 17;
 }
+
+// Required parameters for probabilistic micropayment tickets
+message TicketParams {
+    // Pay out to the recipient if the ticket wins
+    bytes face_value = 1;
+
+    // Probability that the ticket wins
+    bytes win_prob = 2;
+
+    // 32 byte keccak-256 hash commitment to a random number provided
+    // by the recipient
+    bytes recipient_rand_hash = 3;
+
+    // Value generated by recipient that the recipient can use
+    // to derive the random number corresponding to the recipient's hash commitment
+    bytes seed = 4;
+}
+
+// Probabilistic micropayment ticket
+message Ticket {
+    // ETH address of the recipient
+    string recipient = 1;
+
+    // ETH address of the sender
+    string sender = 2;
+
+    // Pay out to the recipient if the ticket wins
+    bytes face_value = 3;
```

Comment:
In wei, right? Would a uint64 [1] work here to avoid a post-processing step, and carrying around a bignum? I guess this technically limits the recipient to ~18 ETH per winning ticket [2] but that seems OK.

[1] https://developers.google.com/protocol-buffers/docs/proto3#scalar
[2] 2^(64-lg(1e18))

Reply:
IMO it seems more cost-effective to pay the price of additional processing (converting `[]byte` to `*big.Int`) for the benefit of avoiding any bugs with people trying to use higher values of `face_value`. It seems to me like an awkward limitation to introduce by design. I believe the performance cost should be negligible.

wdyt?

Reply:
Yeah, I also find it slightly annoying to carry around a `*big.Int` everywhere to indicate the price, but it's not that big of a deal.

Also wonder if we should still have reasonable limits for safety, especially since we're dealing in wei -- I can see someone accidentally tacking on a couple too many zeros and having an "oh crap" moment when the ticket is submitted. But that's probably besides this particular discussion (and still somewhat unlikely since O sets the price; B would simply not use an O with a too-high price and O doesn't have visibility into B's preferences).